src/itest/java/uk/gov/companieshouse/disqualifiedofficers/search/consumer/TestData.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.disqualifiedofficers.search.consumer;
+
+import org.springframework.util.FileCopyUtils;
+import uk.gov.companieshouse.stream.EventRecord;
+import uk.gov.companieshouse.stream.ResourceChangedData;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class TestData {
+
+    public static final String CHANGED = "changed";
+    public static final String CONTEXT_ID = "context_id";
+    public static final String RESOURCE_ID = "12345678";
+    public static final String RESOURCE_KIND = "disqualified-officers";
+    public static final String CHARGES_RESOURCE_URI = "/disqualified-officers/12345678";
```

Comment:
Should this be disqualified-officers/natural/12345678 and the name reference disqs?